### ISSUE TITLE
Feature [GitHub Externship]:

### ISSUE THREAD

**Issue by mahajan1217** [hoppscotch/hoppscotch#1637]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. Ex. I'm always frustrated when I am not able to link third-party tools or SAAS applications using API because there is no backend in that application to do that. Suppose I am working on WordPress and I want to include a third-party website but the company is too new that they not integrated it yet, or WordPress doesnt have an option to do that as they got no connection with the company.

**Describe the solution you'd like**
There can be a feature where we can link anything to anything. For eg I want to include Airtable in my application but it's difficult or sometimes this feature is not available. Boom no worries we made that tool and can help you with that.
So basically we can link our application from one end and then link the service we want to use from another end and the user doesn't have to do anything.

**Describe alternatives you've considered**

**Additional context**

**Comment by liyasthomas** [comment]
@mahajan1217

I'm not sure if "Issues" is the right place for you to add this suggestion. The "Issues" section is strictly reserved for bugs and feature requests directly related to Hoppscotch web client.

If you'd like to propose a GitHub externship proposal, please submit them in "[GitHub Externship](https://github.com/hoppscotch/hoppscotch/discussions/categories/github-externship)" discussions category.

Please use [this template](https://github.com/hoppscotch/hoppscotch/discussions/1616) while submitting the proposal.

Feel free to ask any doubts in our [Discord server](https://hoppscotch.io/discord) or [Telegram group](https://hoppscotch.io/externship_telegram).
